packages/dialtone-vue2/recipes/conversation_view/attachment_carousel/attachment_carousel.stories.js

```diff
@@ -0,0 +1,131 @@
+import { action } from '@storybook/addon-actions';
+import { createRenderConfig } from '@/common/storybook_utils';
+import DtRecipeAttachmentCarousel from './attachment_carousel.vue';
+import DtRecipeAttachmentCarouselDefaultTemplate from './attachment_carousel_default.story.vue';
+
+/*
+  Controls
+  ========
+
+  Here we define any custom configuration for props / slots / events in storybook
+
+  By default, storybook will display any props / slots / events from the associated component. It will also use jsdoc
+  comments on the component to populate details such as description and default value. You should only enter config
+  here if it was not possible to add into the jsdoc of the component itself.
+
+  See https://storybook.js.org/docs/vue/api/argtypes#manual-specification
+
+  To set the description of a slot, add the below comment to the line above where the slot is defined:
+  <!-- @slot example slot decorator -->
+*/
+
+const mediaList = [
+  {
+    type: 'image',
+    path: 'https://vue.dialpad.design/assets/test-078acfea.jpg',
```

Comment:
this is referencing vue.dialpad.design so the images aren't showing up.. guessing this is a problem on the vue 3 branch also.

Reply:
Changed to a relative url, should work correctly now on vue 2 and vue 3